monkestation/code/modules/antagonists/living_lube/living_lube_event.dm

```diff
@@ -0,0 +1,32 @@
+/datum/round_event_control/living_lube
+	name = "Ghost of Honks Past"
+	typepath = /datum/round_event/ghost_role/living_lube
+	weight = 2 //don't want this little lube appearing Too often
+	max_occurrences = 1
+
+/datum/round_event/ghost_role/living_lube
+	minimum_required = 1
+	role_name = "Ghost of Honks Past"
+	fakeable = FALSE
+
+/datum/round_event/ghost_role/living_lube/spawn_role()
+	var/list/candidates = get_candidates()
+
+	if(!candidates.len)
+		return NOT_ENOUGH_PLAYERS
+
+	var/mob/dead/selected = pick_n_take(candidates)
+	var/mob/living/simple_animal/hostile/retaliate/clown/lube/living_lube = new(pick(GLOB.xeno_spawn))
+
+	var/datum/mind/new_mind = new /datum/mind(selected.key)
```

Comment:
Modify the mind rather than creating a new one

Reply:
Every other antag makes a new one, and I think its because you can't modify the assigned_role otherwise it just puts a runtime in the logs and returns. Or at least that's what happened when I did it, not sure if its anything else that could've gone wrong.